cmd/clusterctl/client/config/reader_viper.go

```diff
@@ -102,6 +121,30 @@ func (v *viperReader) Init(path string) error {
 	return nil
 }
 
+func downloadFile(url string, filepath string) error {
+	// Create the file
+	out, err := os.Create(filepath)
+	if err != nil {
+		return err
```

Comment:
might be `err.Wrapf` to provide some context for errors?

Reply:
done. Thanks so much for your review!